slave-base-centos7/Dockerfile

```diff
@@ -2,16 +2,18 @@ FROM centos:centos7
 
 MAINTAINER Adam Saleh <asaleh@redhat.com>
 
-ENV HOME=/home/jenkins
+ENV HOME=/home/jenkins \
+    ENABLED_COLLECTIONS="rh-git29"
 
 USER root
 
-RUN yum install -y https://centos7.iuscommunity.org/ius-release.rpm && \
-    yum install -y centos-release-scl-rh && \
-    INSTALL_PKGS="bc expect gettext git2u httpd-tools java-1.8.0-openjdk-headless lsof rsync tar unzip which zip gcc" && \
+RUN yum install -y centos-release-scl-rh && \
+    INSTALL_PKGS="bc expect gettext rh-git29-git httpd-tools java-1.8.0-openjdk-headless lsof rsync tar unzip which zip gcc" && \
     yum install -y --setopt=tsflags=nodocs install $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
     yum clean all && \
+    rm -rf /var/cache/yum && \
+    source scl_source enable rh-git29 && \
```

Comment:
Actually I think this `source scl_source enable rh-git29` line can be removed, now that the `slave-base-centos7/contrib/bin/configure-slave` file is there. I'll make sure, and then remove it here if I can.

Reply:
This only needs to be sourced in here if you are actually going to run some git command in the Dockerfile. How is configure-slave being sourced on container start though?